docs/cue/reference/components/sinks/aws_sqs.cue

```diff
@@ -89,7 +89,24 @@ components: sinks: aws_sqs: components._aws & {
 			type: string: {
 				default: null
 				examples: ["vector", "vector-%Y-%m-%d"]
-				syntax: "literal"
+				syntax: "template"
```

Comment:
This was just incorrectly documented?

Reply:
Ah, yeah, I meant to call that out in the PR. That field should be marked as a template.

Reply:
That's what I gathered based on the examples and existing 😄 no worries